target/linux/lantiq/files-5.4/arch/mips/boot/dts/lantiq/vr9_arcadyan_vg3503j.dts

```diff
@@ -56,62 +56,50 @@
 	};
 };
 
-&eth0 {
+&gphy0 {
```

Comment:
GPHY_MODE_GE is set by default. Maybe we can remove it?

Reply:
there's no ``lantiq,gphy-mode = <GPHY_MODE_GE>;`` in vr9.dtsi
so I'm not sure where that default you mentioned is coming from

Reply:
If 'lantiq,gphy-mode' is missing, driver set by default GPHY_MODE_GE.
https://elixir.bootlin.com/linux/latest/source/drivers/net/dsa/lantiq_gswip.c#L1721

Reply:
> If 'lantiq,gphy-mode' is missing, driver set by default GPHY_MODE_GE.

interestingly this is not documented at all in the dt-bindings:
https://elixir.bootlin.com/linux/v5.8.7/source/Documentation/devicetree/bindings/net/dsa/lantiq-gswip.txt

personally I feel that making the property mandatory is a good thing to have because it makes the final .dts files easier to read. this is important because after all these are written by humans (compared to other .dts files where parts are generated by the bootloader).
what do you think? depending on the outcome of this discussion I'll send a fix for the dt-bindings upstream